diff --git a/Makefile b/Makefile
index 2121571..922b7bd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 EFIFILES = HelloWorld.efi LockDown.efi Loader.efi ReadVars.efi UpdateVars.efi \
-	KeyTool.efi HashTool.efi SetNull.efi ShimReplace.efi
+	KeyTool.efi HashTool.efi SetNull.efi ShimReplace.efi AutoEnroll.efi
 BINARIES = cert-to-efi-sig-list sig-list-to-certs sign-efi-sig-list \
 	hash-to-efi-sig-list efi-readvar efi-updatevar cert-to-efi-hash-list \
 	flash-var
@@ -93,6 +93,7 @@ ReadVars.so: lib/lib-efi.a lib/asn1/libasn1-efi.a
 UpdateVars.so: lib/lib-efi.a
 LockDown.so: lib/lib-efi.a
 KeyTool.so: lib/lib-efi.a lib/asn1/libasn1-efi.a
+AutoEnroll.so: lib/lib-efi.a lib/asn1/libasn1-efi.a
 HashTool.so: lib/lib-efi.a
 PreLoader.so: lib/lib-efi.a
 HelloWorld.so: lib/lib-efi.a
